modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -54,15 +55,16 @@ exports.update = function (req, res) {
 exports.changeProfilePicture = function (req, res) {
   var user = req.user;
   var message = null;
+  var upload = multer({ dest: './modules/users/client/img/profile/uploads/' }).single('newProfilePicture');
 
   if (user) {
-    fs.writeFile('./modules/users/client/img/profile/uploads/' + req.files.file.name, req.files.file.buffer, function (uploadError) {
-      if (uploadError) {
+    upload(req, res, function (err){
+      if(err) {
         return res.status(400).send({
           message: 'Error occurred while uploading profile picture'
         });
       } else {
-        user.profileImageURL = 'modules/users/client/img/profile/uploads/' + req.files.file.name;
+        user.profileImageURL = req.file.path;
```

Comment:
Thanks to @bastianwegge for pointing this out..

This line will not work in Windows with the current implementation of Multer, because of how it is storing the path. It's not accommodating for Windows paths. 

Can you update this line to use the following instead?
`user.profileImageURL = 'modules/users/client/img/profile/uploads/' + req.file.filename;` 

I've tested this and it works in Firefox as well.